source/directives/rhForm.js

```diff
@@ -43,6 +44,17 @@ module.exports = [ "$compile", function ( $compile )
                         scope.save( scope.model );
                     }
                 };
+                scope.getShowDeleteState = function ()
+                {
+                    // NOTE: Maintain Default for Previous Implementation
+                    if ( !(typeof( scope.showDelete ) === 'boolean' || angular.isFunction( scope.showDelete )) )
+                    {
+                        return true;
+                    }
```

Comment:
It was my attempt at limiting the `rhForm` directive argument `showDelete` to either a (`Function` or `Boolean`), and returning `true` as a default fallback